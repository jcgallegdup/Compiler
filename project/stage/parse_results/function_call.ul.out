Insert key=Implement me! val=AST.Function@76ed5528
Error:0(line):0(position): Could not find function with name 'main' and type 'void'.
