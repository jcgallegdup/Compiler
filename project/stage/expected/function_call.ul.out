void m()
{
    id(expr);
    id(expr('));
    id(((expr(""))));
    id(id(((expr(1)))));
    ___VALID_ID(((arg1)), "anotherArg123", IDENTIFIER1[funcCall(0, ')], RATTLED(true, false, "MEEP"));
}
